### DISCUSSION TITLE
"Accuracy" shown in the break time should use the same string as in the score panel

### DISCUSSION THREAD

**Discussion by refparo** [ppy/osu#17986] [Q&A] [answered]
Currently it is using the same string as in beatmap details. These two strings mean different things despite both being read as "accuracy", and they are translated differently into other languages.

![图片](https://user-images.githubusercontent.com/13145126/165311418-c9b229b5-06b6-459a-aec8-3646d0a12d27.png)
![图片](https://user-images.githubusercontent.com/13145126/165311526-845999a5-433f-43d1-bc20-fdd563b87cde.png)
↑"accuracy" in the break time. The Chinese translation is wrong in this context.

![图片](https://user-images.githubusercontent.com/13145126/165310437-c6e8a7f6-4a0f-4465-b6cb-56e0da38a8fd.png)
![图片](https://user-images.githubusercontent.com/13145126/165311957-e21066db-12ad-41c4-ac82-99abaf8cd57f.png)
↑"accuracy" in beatmap details. The Chinese translation here literally means "accuracy requirement (of the beatmap)", which is not suitable for the context in break time.

![图片](https://user-images.githubusercontent.com/13145126/165311865-75977664-757a-4906-aba9-91fbef595402.png)
![图片](https://user-images.githubusercontent.com/13145126/165311806-6965b8f3-09cb-45b8-80de-b89f1b68425f.png)
↑"accuracy" in the score panel. The Chinese translation here literally means "accuracy". This string should be used for that "accuracy" in the break time.

**Comment by frenzibyte** [comment, marked as answer]
Should be resolved in #17988
